apps/api/src/app/modules/sessions/sessions.service.ts

```diff
@@ -62,10 +62,19 @@ export class SessionsService implements OnModuleInit {
       },
     };
 
-    const sessionMatch: { [key: string]: any } = {};
+    const sessionMatch: { [key: string]: unknown } = {};
 
     if (_id) {
-      sessionMatch._id = { $regex: _id, $options: 'i' };
+      sessionMatch['_id'] = _id;
+    }
+
+    const sessionsSort: { [key: string]: 1 | -1 } = {};
```

Comment:
nitpick: `sessionSort` instead of `sessionsSort`. Adjectives (even nominal ones) are invariable in english.

Reply:
Ah, c'est bon à savoir !

Done